src/main/java/com/demo/MybatisApplication/exceptions/ResourceNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.demo.MybatisApplication.exceptions;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
```

Comment:
If we have an exceptionHandler then do we need this annotation?
Or if we have this annotation then do we need exception handler?

Reply:
No they are not dependent on each other.